src/hotspot/share/gc/parallel/parallelScavengeHeap.hpp

```diff
@@ -119,8 +119,7 @@ class ParallelScavengeHeap : public CollectedHeap {
     _survivor_pool(NULL),
     _old_pool(NULL),
     _workers("GC Thread",
-             ParallelGCThreads,
-             false /* are_ConcurrentGC_threads */) { }
+             ParallelGCThreads) { }
```

Comment:
Consider moving this up to the line above, like the G1 code.

Reply:
Fixed.